packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -166,13 +158,13 @@ function registerCommands(
     ORG_OPEN_COMMAND,
     forceOrgOpen
   );
-  const forceSourceDeleteCmd = vscode.commands.registerCommand(
-    'sfdx.force.source.delete',
-    forceSourceDelete
+  const projectDeleteSourceCmd = vscode.commands.registerCommand(
```

Comment:
Earlier when we were shipping extensions as a wrapper around the CLI commands. Now we are using libraries to execute deploy, retrieve etc. I think we don't need to map the variable name with commands as it is. We can just skip using force or project. 
We can use variable names like `DeleteSourceCmd`.